### ISSUE TITLE
Split breaks when bundled with webpack

### ISSUE THREAD

**Issue by tgroutars** [splitio/javascript-client#567]
We are working with split, and use it accross several projects. It's a dependency of one of core packages, and we include that package in several places, including lambdas.

We use webpack to bundle the lambdas because we need to include our internal dependency in the bundle. The issue we are getting is when we also include `@splitsoftware/splitio` inside the bundle. Everything else works fine but split doesn't play nicely with that approach.

Here's the error message when we run the lambda and it tries to use split
```
  TypeError: (0 , _getFetch__WEBPACK_IMPORTED_MODULE_0__.default)(...) is not a function
      at Fetcher (webpack://sl-api/../../packages/maze-api-core/node_modules/@splitsoftware/splitio/es/services/transport/index.js?:16:62)
      at splitChangesFetcher (webpack://sl-api/../../packages/maze-api-core/node_modules/@splitsoftware/splitio/es/producer/fetcher/SplitChanges.js?:36:86)
      at splitChanges (webpack://sl-api/../../packages/maze-api-core/node_modules/@splitsoftware/splitio/es/producer/updater/SplitChanges.js?:78:90)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

I've been digging a bit and it seems like the `getFetch` directory doesn't contain an `index.js`, but a `package.json`, and it seems like webpack cannot resolve this, at least not on its own. 

Is this a bug or should I include something in our webpack config to support it? Any pointer would be appreciated 🙏 

The version we are running is `@splitsoftware/splitio@10.15.2`

Thanks in advance!

**Comment by chillaq** [comment]
Hi @tgroutars , could you please provide the package.json file? Just to get all versions used in your app.

Thanks
Bilal

**Comment by tgroutars** [comment]
@chillaq Thanks for the quick response! I'm afraid this might be sentitive info, but I'll try to provide a minimal repro!

**Comment by EmilianoSanchez** [comment]
Hi @tgroutars 
Your issue seems to be the same than this one #505 , for which we provided a fix in `@splitsoftware/splitio@10.15.3`.
I would try upgrading your split dependency to that version at least.

Thanks, 
Emiliano

**Comment by tgroutars** [comment]
That did indeed fix it, thank you so much! I thought we were on the latest version, but we were on `10.15.2`
